LeetCode/Array/Search_in_Rotated_Sorted_Array/HyeonJoeng.py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    def search(self, nums: List[int], target: int) -> int:
+        if target not in nums:
+            return -1
+        answer = [i for i, x in enumerate(nums) if x == target]
+        return int(answer[0])
```

Comment:
```suggestion
        return int([i for i, x in enumerate(nums) if x == target][0])
```
억지로 줄이겠다면, 이런식으로 할 수 있겠죠?
근데 지금 처럼 표현하는게 더 좋을 것 같아요~ 잘 풀이하셨습니다!